src/network/MissionControlProtocol.cpp

```diff
@@ -176,6 +177,34 @@ void MissionControlProtocol::handleJointPowerRequest(const json& j) {
 	}
 }
 
+void MissionControlProtocol::sendRoverPos() {
+	auto heading = robot::readIMUHeading();
+	auto gps = robot::readGPS();
+	if (gps.isValid() && heading.isValid()) {
+		double orientX = 0.0;
+		double orientY = 0.0;
+		double orientZ = sin(heading.getData() / 2);
+		double orientW = cos(heading.getData() / 2);
+		double posX = gps.getData()[0];
+		double posY = gps.getData()[1];
+		double posZ = 0.0;
+		double recency = util::durationToSec(dataclock::now() - gps.getTime()) >
+								 util::durationToSec(dataclock::now() - heading.getTime())
+							 ? util::durationToSec(dataclock::now() - gps.getTime())
+							 : util::durationToSec(dataclock::now() - heading.getTime());
```

Comment:
This statement is way too long. Can you put all the `util::durationToSec` calls into variables, and then just simplify this by calling `std::max`?

I.e.:
```c++
double gpsRecency = util::durationToSec(...);
double headingRecency = util::durationToSec(...);
double recency = std::max(gpsRecency, headingRecency);
```
Which is both easier to read and shorter.